.azure-pipelines/release-test.yml

```diff
@@ -1,178 +1,174 @@
 pr: none
 trigger: none
 
-strategy:
-  matrix:
-    Windows PS 5.1.14:
-      Image_Name: 'vs2017-win2016'
-      PSVersion: '5.1.14'
-  
-    Windows PS 6.2.4:
-      Image_Name: 'vs2017-win2016'
-      PSVersion: '6.2.4'
-     
-    Windows PS 7.0.3:
-      Image_Name: 'vs2017-win2016'
-      PSVersion: '7.0.3'
-
-    Linux PS 6.2.4:
-      Image_Name: 'ubuntu-18.04'
-      PSVersion: '6.2.4'
+jobs:
+- job: SmokeTest
+  timeoutInMinutes: 180  
+  strategy:
+    matrix:
+      Windows PS 5.1.14:
+        Image_Name: 'vs2017-win2016'
+        PSVersion: '5.1.14'  
+      Windows PS 6.2.4:
+        Image_Name: 'vs2017-win2016'
+        PSVersion: '6.2.4'     
+      Windows PS 7.0.3:
+        Image_Name: 'vs2017-win2016'
+        PSVersion: '7.0.3'
+      Linux PS 6.2.4:
+        Image_Name: 'ubuntu-18.04'
+        PSVersion: '6.2.4'      
+      Linux PS 7.0.3:
+        Image_Name: 'ubuntu-18.04'
+        PSVersion: '7.0.3'  
+      MacOS PS 6.2.4:
+        Image_Name: 'macOS-10.15'
+        PSVersion: '6.2.4'
+      MacOS PS 7.0.3:
+        Image_Name: 'macOS-10.15'
+        PSVersion: '7.0.3'
+  pool:
+    vmImage: $(Image_Name)
+
+  steps:
+  - task: PowerShell@2
+    displayName: Prepare Powershell $(PSVersion) 
+    inputs:
+      filePath: 'tools/Test/SmokeTest/PrepareRequiredPowershell.ps1'
+      arguments: '-RequiredPsVersion $(PSVersion)'      
       
-    Linux PS 7.0.3:
-      Image_Name: 'ubuntu-18.04'
-      PSVersion: '7.0.3'
-  
-    MacOS PS 6.2.4:
-      Image_Name: 'macOS-10.15'
-      PSVersion: '6.2.4'
-
-    MacOS PS 7.0.3:
-      Image_Name: 'macOS-10.15'
-      PSVersion: '7.0.3'
-
-pool:
-  vmImage: $(Image_Name)
-
-steps:
-- task: PowerShell@2
-  displayName: Prepare Powershell $(PSVersion) 
-  inputs:
-    filePath: 'tools/Test/SmokeTest/PrepareRequiredPowershell.ps1'
-    arguments: '-RequiredPsVersion $(PSVersion)'      
+  - task: DownloadPipelineArtifact@2
+    condition: and(succeeded(), eq(variables['GalleryName'], 'LocalRepo'))
+    displayName: Download Artifacts from Build Pipeline
+    inputs:
+      buildType: 'specific'
+      project: $(ProjectToDownloadArtifacts)
+      definition: $(BuildPipelineName)
+      buildVersionToDownload: 'specific'
```

Comment:
Is it possible to download latest build?
or is it necessary/useful/helpful to download latest build?

Reply:
1. Sure, here is a test run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=581795&view=results
2. It is possible to download any build you want to. If anyone want to download latest build, fill its pipeline instance ID into the variable `PipelineId` before run.